src/backend/be_utils.js

```diff
@@ -0,0 +1,35 @@
+const BD_TRANSACAO = "";
+
+const json_test = {
+        "_id": "1",
+        "data": "Thu Jun 21 2023 09:54:51 GMT-0300",
+        "tipo": "cashback",
+        "db": "15,60",
+        "nome": "Posto Machado"
+}
+
+const json_test2 = {
+    "_id": "2",
+    "data": "Thu Jun 22 2023 09:54:51 GMT-0300",
+    "tipo": "pagamento_cartao",
+    "db": "242,50",
+    "nome": "Posto Campestre"
+}
+
+const json_test3 = {
+    "_id": "3",
+    "data": "Thu Jun 23 2023 09:54:51 GMT-0300",
+    "tipo": "pagamento_dinheiro",
```

Comment:
there is no "pagamento_cartao" and "pagamento_dinheiro" atm, they are the same, payments are not yet done in the app